cordova-lib/spec-plugman/platforms/ios.spec.js

```diff
@@ -301,14 +301,8 @@ describe('ios project handler', function() {
             });
         });
         it('of two plugins should apply xcode file changes from both', function(){
```

Comment:
I think you should add `done` here and call it at the end of the promise. See what happens. 

You can also chain the two promises (move 315 into 313). 

Add some console logs to confirm these tests are actually running